CMakeLists.txt

```diff
@@ -167,8 +167,16 @@ add_feature_info("elogind" ELOGIND_FOUND "elogind support")
 
 # Default behaviour if neither systemd nor elogind is used
 if (NOT ELOGIND_FOUND AND NOT SYSTEMD_FOUND)
-    set(MINIMUM_VT 7)
-    set(HALT_COMMAND "/sbin/shutdown -h -P now")
+    # Set the VT on which sddm will normally appear, and the
+    # commands for shutdown and reboot. On FreeBSD, there are
+    # normally more getty's running than on Linux.
+    if(${CMAKE_SYSTEM} MATCHES "FreeBSD")
```

Comment:
i think you should quote "${CMAKE_SYSTEM}"

Reply:
Also use CMAKE_SYSTEM_NAME instead of CMAKE_SYSTEM, the former might include the version